scripts/install/get_tendermint.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#set -x
+
+# an examplary download-url
+# https://github.com/tendermint/tendermint/releases/download/v0.34.13/tendermint_0.34.13_linux_amd64.tar.gz
+export TM_MAJORMINOR="0.34"
+export TM_PATCH="13"
+export TM_REPO="https://github.com/tendermint/tendermint"
+
+export TM_VERSION="${TM_MAJORMINOR}.${TM_PATCH}"
+
+export TARGET_PATH="/usr/local/bin"
+export TMP_PATH="/tmp"
+
+error_exit()
+{
+    echo "Error: $1"
+    exit 1
+}
+
+# check for existence
+export TM_EXISTS_VER=$(tendermint version)
```

Comment:
This fails with `scripts/install/get_tendermint.sh: line 23: tendermint: command not found`, maybe we could silence it

Reply:
hello tomas,
 
sure ... this problem should be fixed with my latest push to the branch.

br,
joachim